TranslatorReasonerAPI.yaml

```
@@ -322,7 +322,7 @@ components:
           type: string
           example: Haptoglobin
           description: Formal name of the entity
-        type:
+        category:
           oneOf:
             - type: string
```

```suggestion
            - $ref: '#/components/schemas/BiolinkEntity'
```
This is just an unrelated fix I noticed during review.